src/simulator_core/entities/assets/controller_classes.py

```diff
@@ -0,0 +1,64 @@
+"""Module containing the classes for the controller."""
+import datetime
+
+import pandas as pd
+
+
+class AssetControllerAbstract:
+    """Abstract class for the asset controller."""
+
+    def __init__(self, name: str, identifier: str):
+        """Constructor for the asset controller.
+
+        :param str name: Name of the controller object.
+        :param str identifier: Unique identifier of the controller asset.
+        """
+        self.name = name
+        self.id = identifier
+
+
+class ControllerConsumer(AssetControllerAbstract):
+    """Class to store the consumer for the controller asset."""
+
+    def __init__(self, name: str, identifier: str):
+        """Constructor for the consumer.
+
+        :param str name: Name of the consumer.
+        :param str identifier: Unique identifier of the consumer.
+        """
+        super().__init__(name, identifier)
+        self.temperature_return = 40 + 273.15
```

Comment:
Waarom zou je het hardcode? Ik zou hier de defaults voor gebruiken dan hoeven we het niet op meer plekken aan te passen